packages/vscode-extensions/src/browser/style/extension-detail.css

```diff
@@ -0,0 +1,115 @@
+/********************************************************************************
+ * Copyright (C) 2019 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.vscode-extension-detail {
```

Comment:
please check that code in PR is aligned with our coding conventions

i.e here naming is bogus: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css-prefix-global-classes